definitions/3.0.0/channelMessages.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I understand we need this because messages are not by default list

if I got it right, just ignore, you just got `rubber-ducked`

Reply:
yup, thanks!